Corona-Warn-App/src/main/res/drawable/ic_submission_home.xml

```diff
@@ -1,12 +1,10 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="28dp"
-    android:height="28dp"
-    android:viewportWidth="28"
-    android:viewportHeight="28">
-    <path
-        android:fillColor="#3C4143"
-        android:fillType="nonZero"
-        android:pathData="M14,6.3636C13.8337,6.3636 13.6739,6.4288 13.555,6.5451L6.6135,12.2227C6.6054,12.2287 6.5976,12.2349 6.5898,12.2413L6.5662,12.2612L6.5662,12.2624C6.4372,12.3826 6.3639,12.5509 6.3636,12.7273C6.3636,13.0787 6.6485,13.3636 7,13.3636L7.6364,13.3636L7.6364,20.3636C7.6364,21.0668 8.2059,21.6364 8.9091,21.6364L19.0909,21.6364C19.7941,21.6364 20.3636,21.0668 20.3636,20.3636L20.3636,13.3636L21,13.3636C21.3515,13.3636 21.6364,13.0787 21.6364,12.7273C21.6364,12.5505 21.5631,12.3817 21.4338,12.2612L21.4238,12.2537C21.4099,12.2411 21.3954,12.229 21.3803,12.2177L19.0909,10.3446L19.0909,7.6364C19.0909,7.2851 18.8058,7 18.4545,7L17.8182,7C17.4669,7 17.1818,7.2851 17.1818,7.6364L17.1818,8.7836L14.4313,6.5339C14.3141,6.4249 14.1601,6.3641 14,6.3636ZM12.0909,15.2727L15.9091,15.2727L15.9091,21.6364L12.0909,21.6364L12.0909,15.2727Z"
-        android:strokeWidth="1"
-        android:strokeColor="#00000000" />
+    android:width="22dp"
+    android:height="20dp"
+    android:viewportWidth="22"
+    android:viewportHeight="20">
+  <path
+      android:pathData="M11,0.5C10.7329,0.5 10.4769,0.6069 10.2891,0.7969L0.2031,9.5977C0.0754,9.6919 -0,9.8412 0,10C0,10.2761 0.2239,10.5 0.5,10.5H3V18.5C3,19.052 3.448,19.5 4,19.5H8C8.552,19.5 9,19.052 9,18.5V12.5H13V18.5C13,19.052 13.448,19.5 14,19.5H18C18.552,19.5 19,19.052 19,18.5V10.5H21.5C21.7761,10.5 22,10.2761 22,10C22,9.8412 21.9246,9.6919 21.7969,9.5977L11.7168,0.8027C11.7149,0.8008 11.7129,0.7988 11.7109,0.7969C11.5231,0.6069 11.2671,0.5 11,0.5Z"
+      android:fillColor="#ffffff"
+      android:fillType="evenOdd"/>
```

Comment:
We already have a home icon `ic_risk_details_home`. Cannot we reuse it?

Reply:
The one from the designs looks different from this one. Also, the current one that we have is too small. Looked out of place.

Reply:
Oh no, wait. This is another home icon :D I think that will work.